### PR TITLE
 Add docs for Reconciliation

### PR DESCRIPTION
**Pull request by yuwenmemon**
@francoisl please review

For https://github.com/Expensify/Expensify/issues/40854

 Add docs for reconciliation. Copied everything over from the old doc: https://integrations.expensify.com/Integration-Server/doc/reconciliation.html

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -234,6 +234,143 @@ See the [export template format reference](./export_report_template.html) for mo
...
 
+## Reconciliation
+
+> - Export unreported transactions on all feeds between 1/1/2016 and 10/10/2016 to a CSV file.
+
+```shell
+{
+    "type":"reconciliation",
+    "credentials":{
+        "partnerUserID":"...",
```
Comment (francoisl) [inline source/index.html.md]: the rest of the docs uses `_REPLACE` for consistency

File: source/index.html.md [hunk 2]
```diff
@@ -234,6 +234,143 @@ See the [export template format reference](./export_report_template.html) for mo
...
+| inputSettings | JSON object | See inputSettings | Settings used to filter the reports that are exported. |
+**Optional elements** |
+| outputSettings | JSON object | See outputSettings | Settings for the generated file.
+
+
+- `inputSettings`
+
+Name | Format | Valid values | Description
+--------- | --------- | --------- | ---------
+| startDate | String | yyyy-mm-dd formatted date | The beginning date for which we will include expenses in the generated report.
```
Comment (francoisl) [inline source/index.html.md]: nab: you don't need the `|` at the beginning

File: source/index.html.md [hunk 3]
```diff
@@ -234,6 +234,143 @@ See the [export template format reference](./export_report_template.html) for mo
...
+</#list>
+```
+> - **Note:** Reconciliation templates follow a slightly different format than Export Templates. The basic iterative skeleton of the template is:
+
+```
+<#list cards as card, reports>
+    <#-- Properties to print per card -->
+    <#list reports as report>
+        <#-- Properties to print per report -->
+        <#list report.transactionList as transaction>
```
Comment (francoisl) [inline source/index.html.md]: we should use `as expense` for consistency, that's how we called the expense in all the other templates from the start.

**Review by francoisl — CHANGES_REQUESTED** [review]

**Comment by yuwenmemon** [comment]
Updated

**Review by francoisl — APPROVED** [review]

**Pull request merged**